components/global/Tito/Tito.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { TitoStyledWrapper } from './Tito.styled'
+import Conference from '../../../config/conference'
```

Comment:
It's better to pass the conference instance in rather than using it statically. That allows for a potential future state where conference data is loaded via ajax from an api, but also allows for components to potentially get used between conference instances (e.g. have an archive page for a previous conference etc.).

Reply:
ah, I had wondered about the two variations. Thanks for clearing that up. I'll bring it in through props instead.